extensions/data-transfer/portability-data-transfer-deezer/src/main/java/org/datatransferproject/transfer/deezer/playlists/DeezerPlaylistExporter.java

```diff
@@ -91,9 +93,10 @@ private PlaylistContainerResource enumeratePlaylists(DeezerApi api)
   private MusicRecording convertTrack(DeezerApi api, long trackId) throws IOException {
     Track track = api.getTrack(trackId);
     return new MusicRecording(
+        track.getLink(),
```

Comment:
Do you want to add the music related changes to the PR description as a subpoint?